src/main/kotlin/com/example/todolist/global/entity/BaseUUIDEntity.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.todolist.global.entity
+
+import org.hibernate.annotations.GenericGenerator
+import java.util.*
+import javax.persistence.Column
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+import javax.persistence.MappedSuperclass
+
+@MappedSuperclass
+abstract class BaseUUIDEntity(
+    @Id
+    @GeneratedValue(generator = "uuid2")
+    @GenericGenerator(name = "uuid2", strategy = "uuid2")
+    @Column(columnDefinition = "BINARY(16)", nullable = false)
+    open val idx: UUID
+): BaseTimeEntity()
```

Comment:
```suggestion
) : BaseTimeEntity()
```
전체적인 코드들은 한칸 띄우고 콜론을 쓰던데 통일 시키는게 좋을거 같습니다.

Reply:
수정했습니다